### ISSUE TITLE
the reserved bit width should be 16

### ISSUE THREAD

**Issue by baimengwei** [riscv-non-isa/riscv-iommu#275]
https://github.com/riscv-non-isa/riscv-iommu/blob/main/iommu_ref_model/libiommu/include/iommu_data_structures.h#L240
```
typedef union {
    struct {
        uint64_t PPN:44; 
        uint64_t reserved:15; 
        uint64_t MODE:4; 
    };
    uint64_t raw;
} msiptp_t;
```

the total width should be 64.

**Comment by ved-rivos** [comment]
Thanks for reporting this @baimengwei . It is fixed in PR #276

**Comment by baimengwei** [comment]
Thanks.
